pkg/services/alerting/eval_context.go

```diff
@@ -99,6 +99,10 @@ func (c *EvalContext) GetNotificationTitle() string {
 
 // GetDashboardUID returns the dashboard uid for the alert rule.
 func (c *EvalContext) GetDashboardUID() (*models.DashboardRef, error) {
+	if c.IsTestRun {
```

Comment:
Need to look closer but I think "IsTestRun" is not for go tests, but rather when someone is using the "Test rule".

So I think when we do this, there should be a dashboard UID.